docs/en/latest/plugins/mqtt-proxy.md

```diff
@@ -23,10 +23,11 @@ title: mqtt-proxy
 
 ## Summary
 
-- [**Name**](#name)
-- [**Attributes**](#attributes)
-- [**How To Enable**](#how-to-enable)
-- [**Delete Plugin**](#delete-plugin)
+- [Summary](#summary)
+- [Name](#name)
+- [Attributes](#attributes)
+- [How To Enable](#how-to-enable)
+- [Delete Plugin](#delete-plugin)
```

Comment:
why we remove `**`?

I see that all the plugin docs follows the original format

Reply:
A lot docs also don't use `**` any longer. So in my opinion it is a general question.

Reply:
Maybe we should use the tool to make uniform changes. I think we can leave it as it is in this PR. 😁

Reply:
agree to @tzssangglass, and @danielkocot Daniel, if not necessary, we'd better keep the Top of Contents and only change doc contents 😄 if the ToC needs to be updated, we'd better update other plugins' docs.

Reply:
Now I found out that happened the markdown plugin in Visual Studio Code removed `**`on saving the file.